java/test/jmri/jmrit/beantable/signalmast/AddSignalMastPanelTest.java

```diff
@@ -14,6 +16,8 @@
 
     @Test
     public void testDefaultSystems() {
+        Assume.assumeFalse(GraphicsEnvironment.isHeadless());
```

Comment:
Typically we don't need to do a headless check for a panel, especially in a constructor test.

JPanels can't be displayed on their own, so they can actually be created on headless systems.

Reply:
Removed!